packages/kotti-ui/source/kotti-style/_variables.scss

```diff
@@ -81,6 +81,7 @@ $size-md: 840px !default;
 $size-lg: 960px !default;
 $size-xl: 1280px !default;
 $size-2x: 1440px !default;
+$size-xxl: 1920px !default;
```

Comment:
do you know if 2x is used anywhere

if xxl needs to be 1920 then you should either make the value of size-2x into 1920 OR you should delete 2x

OR you keep 2x and make xxl --> 3x

Please talk to Alberto about this and involve me if needed

Reply:
@carsoli 2x is used in ampi.
I changed xxl to 3x.